Gemfile

```diff
@@ -28,8 +28,13 @@ gem "webpacker", "~> 4.0"
 
 group :development, :test do
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
+  gem "database_cleaner"
```

Comment:
gem này có tác dụng gì e

Reply:
Dạ Gem này dùng để xóa sạch toàn bộ cơ sở dữ liệu sau mỗi lần chạy test, sau khi chạy xong một test nó sẽ rollback database lại trạng thái trước khi chạy test.